src/sphinx/dev/roadmap.rst

```diff
@@ -5,82 +5,19 @@
 Roadmap
 =======
 
-Goal
-----
+In the following there is an incomplete list about features we at least intend to implement for upcoming Scala IDE versions. Additional features are always possible, beside from performance improvements and bug fixes.
```

Comment:
I've place a bit more detailed header in this doc https://docs.google.com/document/d/1ZD_RE491bedDyEfYTs01mZzD41j_WC8RytdNu-w-Z_4/edit# - I think we could take the one from the google doc